assets/responses/farmsmart_chat_ui_flow.json

```diff
@@ -0,0 +1,344 @@
+{
+  "uid": "1",
+  "title": "Chatbot",
+  "subtitle": "Welcome to FarmSmart chatbot!",
+  "origin": "com.wearemobilefirst.farmsmart",
+  "formResponse": {
+    "type": "com.google.firebase.database",
+    "uri": "profiles/%UserID/",
+    "body": {
+      "name": "%Name",
+      "skillLevel": "%SkillLevel",
+      "farmLocation": "%FarmLocation",
+      "agroEcoZone": "%AgroEcoZone",
+      "landSize": "%LandSize",
+      "motivation": "%Motivation",
+      "peopleInHousehold": "%PeopleInHousehold",
+      "soilType": "%SoilType",
+      "startDate": "%StartDate",
+      "usingIrrigation": "%UsingIrrigation",
+    }
+  },
+  "items": [
+    {
+      "text": "Hi! I'm the FarmSmart chatbot, Jata",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "I'm here to set up your profile and recommend the best crops for your farm",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "What is your name?",
+      "inputRequest": {
+        "type": "com.wearemobilefirst.String",
+        "uri": "%Name",
+        "localStore": false,
+        "inline": true,
+        "title": "Your Name",
+        "validationRegex": "^[a-zA-Z]+$",
+        "responseText": "My name is %Name",
+        "optional": false
+      }
+    },
+    {
+      "text": "Nice to meet you %Name",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "Now I'll ask a few questions to help me recommend the best crops for you",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "What is your farming skill level?",
+      "inputRequest": {
+        "type": "com.wearemobilefirst.MultiChoice",
+        "inline": true,
+        "uri": "%SkillLevel",
+        "title": "Skill Level",
+        "localStore": true,
+        "responseText": "",
+        "optional": false,
+        "args": {
+          "maxSelection": 1,
+          "options": [
+            {
+              "id": "OptionKey1",
+              "title": "Beginner",
+              "description": "No experience with farming",
+              "responseText": "I have no experience farming"
+            },
+            {
+              "id": "OptionKey2",
+              "title": "Intermediate",
+              "description": "Some experience farming",
+              "responseText": "I have farming knowledge"
+            },
+            {
+              "id": "OptionKey3",
+              "title": "Advanced",
+              "description": "Experienced farmer",
+              "responseText": "I have experience farming"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "text": "Where is your farm?",
+      "inputRequest": {
+        "type": "com.wearemobilefirst.Dropdown",
+        "uri": "%FarmLocation",
+        "title": "Farm Location",
+        "localStore": false,
+        "responseText": "",
+        "inline": true,
+        "validationRegex": "^[a-zA-Z]+$",
```

Comment:
Please remove this regexp as in this case we will NOT use it.

Reply:
Done